src/Stack/FileDigestCache.hs

```diff
@@ -0,0 +1,24 @@
+module Stack.FileDigestCache
+  ( FileDigestCache
+  , newFileDigestCache
+  , readFileDigest
+  ) where
+
+import           Stack.Prelude
+import qualified Data.Map.Strict as Map
+import qualified Pantry.SHA256 as SHA256
```

Comment:
Not a substantive point, but other Stack modules list imports alphabetically.

Reply:
adressed.